packages/aws-cdk-lib/aws-s3/lib/bucket.ts

```diff
@@ -964,7 +964,7 @@ export abstract class BucketBase extends Resource implements IBucket {
    * Function to add required permissions to the destination bucket for cross account
    * replication. These permissions will be added as a resource based policy on the bucket
    * @see https://docs.aws.amazon.com/AmazonS3/latest/userguide/replication-walkthrough-2.html
-   * If owner of the bucket needs to be overriden, set accessControlTransition to true and provide
+   * If owner of the bucket needs to be overridden, set accessControlTransition to true and provide
```

Comment:
I see other 5 occurrences of this typo in the project. Can you please adjust those as well?